### PR TITLE
DZ2: Pinentry

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers New pinentry:

screen shot 2016-03-01 at 11 28 33 am

Also updates the login page to share the new larger text area for FormWithCheckbox, and sets the default on the Go side for paper keys to be "Show Typing" true. @patrickxb

### CODE REVIEW

File: shared/util/feature-flags.js [hunk 1]
```diff
@@ -8,23 +8,28 @@ import getenv from 'getenv'
...
   'login': boolean,
-  'mobileAppsExist': boolean
+  'mobileAppsExist': boolean,
+  'dz2': boolean
 }
 
 let features = getenv.array('KEYBASE_FEATURES', 'string', '')
 
-const tracker2 = features.includes(tracker2Key)
+// dz2 implies tracker2
```
Comment (cjb) [inline shared/util/feature-flags.js]: Is this okay? I'd like us to be able to turn on "dz2" (tracker+pinentry) with one feature flag so we can ship that change.

File: shared/common-adapters/text.desktop.js [hunk 2]
```diff
@@ -203,6 +203,19 @@ export const styles = {
     lineHeight: '19px',
     letterSpacing: '0.3px'
   },
+  textBodyBig: {
```
Comment (chrisnojima) [inline shared/common-adapters/text.desktop.js]: this style doesn't exist in the style guide?

Comment (chrisnojima) [inline shared/common-adapters/text.desktop.js]: if this is for the input (under special styles) it appears as 29px line height to me in zeplin. also i think if these are used in these special cases we should maybe put them somewhere else in this file like some 'special' export or something

**Comment by chrisnojima** [comment]
I know these are input problems but
- [ ] Floating text 'passphrase' font off
- [ ] Continue button margins off
- [ ] x button margin off (in header component)
- [ ] Blue underline while focused is too thick

screen shot 2016-03-01 at 12 20 43 pm

**Comment by cjb** [comment]
Okay, all of the pixel tweaks are done now:

screen shot 2016-03-01 at 3 15 02 pm
screen shot 2016-03-01 at 3 06 14 pm

File: shared/common-adapters/text.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline shared/common-adapters/text.desktop.js]: @cjb thoughts on this?

**Comment by chrisnojima** [comment]
Visually close enough for now :+1: would like to discuss the text style thing

File: go/libkb/passphrase_helper.go [hunk 3]
```diff
@@ -40,6 +40,8 @@ func GetPaperKeyPassphrase(ui SecretUI, username string) (string, error) {
 	arg.Prompt = fmt.Sprintf("Please enter a paper backup key passphrase for %s", username)
 	arg.Features.StoreSecret.Allow = false
 	arg.Features.StoreSecret.Readonly = true
+	arg.Features.ShowTyping.Allow = true
+	arg.Features.ShowTyping.DefaultValue = true
```
Comment (MarcoPolo) [inline go/libkb/passphrase_helper.go]: So we default to show typing?

Comment (cjb) [inline go/libkb/passphrase_helper.go]: This is `getPaperKeyPassphrase()`, so we're only defaulting to show typing for paper keys, 'cause they're impossible to type blind.

Comment (MarcoPolo) [inline go/libkb/passphrase_helper.go]: ah okay!

**Comment by MarcoPolo** [comment]
:shipit:

**Pull request merged**